.changes/next-release/bugfix-ManagedUpload-2bd31158.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "ManagedUpload",
+  "description": "fix a bug that credentials refresh to frequently"
```

The type already has bugfix
```suggestion
  "description": "Use resolved credentials if customer supplies configured S3 Client"
```